README.md

```diff
@@ -7,6 +7,17 @@ A python implementation of the UA Parser (https://github.com/ua-parser, formerly
 
 ##Installing
 
+### Install via pip
+First, check that you've got the correct dependencies. For example, on Ubuntu:
+```
+$ apt-get install libyaml libyaml-dev python-dev
```

Comment:
And lastly, looking at how we use `pyyaml`, we don't even attempt to use the c bindings from libyaml anyways.

If we did, we'd have to explicitly do something like, `yaml.load(..., Loader=CSafeLoader)`

`pyyaml` doesn't detect the libyaml bindings automatically, unfortunately. :(